src/pg/connection.cr

```diff
@@ -1,148 +1,56 @@
 require "../pq/*"
 
 module PG
-  class Connection
-    # Connect with all default paramaters.
-    #
-    # This attempts to locate a Unix socket, then falls back to localhost. It
-    # uses your current username as both the user and database.
-    def initialize
-      initialize(PQ::ConnInfo.new)
-    end
+  class Connection < ::DB::Connection
+    protected getter connection
 
-    # Connect with a `PQ::ConnInfo`.
-    #
-    # If you are making a lot of connections to the same database, it may be
-    # useful to reuse the `PQ::ConnInfo` to avoid reparsing the connection
-    # every time. All other initialization methods end up here.
-    def initialize(conninfo : PQ::ConnInfo)
-      @pq_conn = PQ::Connection.new(conninfo)
-      @pq_conn.connect
-      exec("SET extra_float_digits = 3")
```

Comment:
I'm not sure if extra_float_digits is still necessary, so leaving this comment until I get confirmation.